### DISCUSSION TITLE
post/dion-interactive-animation-2/

### DISCUSSION THREAD

**Discussion by giscus** [kakaopay/techlog-comments#79] [Announcements]
# post/dion-interactive-animation-2/

라이브러리 사용 없이 수학과 물리를 이용해 카카오페이에서 진행했던 트리를 밝혀줘 웹 애니메이션 개발 과정을 공유합니다.

https://tech.kakaopay.com/post/dion-interactive-animation-2/

**Comment by pay-dan-dy** [comment]
재밌고 유익한 글 감사합니다!

**Comment by moong23** [comment]
안녕하세요! 글 잘 읽었습니다 :)

중간에 조건식에서 오타가 있느것 같아요!
AS IS:
(y1 < yp && < yp < y2) || (y2 < yp && < yp < y1)
TO BE:
(y1 < yp && yp < y2) || (y2 < yp && yp < y1)

**Reply by dion-hoo** [reply]
감사합니다~ 수정 하도록 하겠습니다🙇
